### PR TITLE
Update travis and setup confs.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length=99
+exclude=dockerfiles
```
Comment (johngian) [inline setup.cfg]: ``flake8`` doesn't check dockerfiles anyway

File: .travis.yml [hunk 2]
```diff
@@ -2,12 +2,11 @@ language: python
 python:
   - "3.6"
 before_install:
-  - "pip install flake8"
-  - "flake8 cis"
-  - "flake8 tests"
+  - pip install flake8
```
Comment (johngian) [inline .travis.yml]: This is exactly the same thing in .yaml

Comment (akatsoulas) [inline .travis.yml]: Agreed, I am just in favor of simplicity. The whole pr is nit.

File: .travis.yml [hunk 3]
```diff
@@ -2,12 +2,11 @@ language: python
 python:
   - "3.6"
```
Comment (johngian) [inline .travis.yml]: If you want to avoid using ``"`` then replace it across all the file.

**Pull request merged**